builders/get-started/quick-start.md

```diff
@@ -0,0 +1,144 @@
+    | Public WSS URLs |                                                                <pre style="padding-right: 2em">```wss://moonriver.public.blastapi.io```</pre> <pre style="padding-right: 2em">```wss://moonriver.api.onfinality.io/public-ws```</pre> <pre style="padding-right: 2em">```wss://moonriver-rpc.dwellir.com```</pre>                                                                |
+
+=== "Moonbase Alpha"
+
+    |    Variable     |                                                                                                                                  Value                                                                                                                                  |
+    |:---------------:|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|
+    |    Chain ID     |                                                                                              <pre style="padding-right: 2em">```{{ networks.moonbase.chain_id }}```</pre>                                                                                               |
+    | Public RPC URLs | <pre style="padding-right: 2em">```https://moonbase-alpha.public.blastapi.io```</pre> <pre style="padding-right: 2em">```https://moonbeam-alpha.api.onfinality.io/public```</pre> <pre style="padding-right: 2em">```https://rpc.api.moonbase.moonbeam.network```</pre> |
+    | Public WSS URLs |  <pre style="padding-right: 2em">```wss://moonbase-alpha.public.blastapi.io```</pre> <pre style="padding-right: 2em">```wss://moonbeam-alpha.api.onfinality.io/public-ws```</pre> <pre style="padding-right: 2em">```wss://wss.api.moonbase.moonbeam.network```</pre>   |
+
+!!! note
+    You can create your own endpoint suitable for development or production from one of the [supported RPC providers](/builders/get-started/endpoints/#endpoint-providers){target=_blank}.
+
+### Block Explorers {: #explorers }
+
+Moonbeam provides two different kind of explorers: ones to query the Ethereum API, and others dedicated to the Substrate API. All EVM-based transactions are accessible via the Ethereum API wheras the Substrate API can be relied upon for Substrate-native functions such as governance, staking, and some information about EVM-based transactions. For more information on each explorer, please check out the [Block Explorers](/builders/get-started/explorers){target=_blank} page.
+
+=== "Moonbeam"
```

Comment:
move link to the URL column instead of the name (fix block explorers page too and make this a snippet)